src/store/actionsV2/__tests__/searchTrialInvestigatorsAction.test.js

```diff
@@ -0,0 +1,35 @@
+import { ACTIVE_TRIAL_STATUSES } from '../../../constants';
+import { searchTrialInvestigatorsAction } from '../searchTrialInvestigatorsAction';
+
+describe('searchTrialInvestigatorsAction', () => {
+	it('should match returned object', () => {
+		const searchText = 'Purple';
+
+		const requestQuery = {
+			agg_field: 'principal_investigator',
+			agg_name: searchText,
+			current_trial_status: ACTIVE_TRIAL_STATUSES,
+			include: ['none'],
+		};
+
+		const expectedAction = {
+			type: '@@api/CTSv2',
+			payload: {
+				service: 'ctsSearchV2',
+				cacheKey: 'tis',
+				method: 'searchTrialInvestigators',
+				requestParams: requestQuery,
+			},
+		};
+
+		expect(searchTrialInvestigatorsAction({ searchText })).toEqual(
+			expectedAction
+		);
+	});
+
+	it('handles exception when called without an searchText parameter', () => {
+		expect(() => {
+			searchTrialInvestigatorsAction({undefined});
```

Comment:
Since this is a named parameter, you'll have to do `searchTrialInvestigatorsAction({searchText: undefined});` or `null` rather than `undefined`.